computing_provider/storage/view.py

```diff
@@ -32,8 +32,11 @@ def process_job(job_data):
 
     job: Job = create_job_from_job_detail(job_data)
     space_name = job.job_source_uri.split('/')[-1]
-    task = build_space_task.delay(space_name)
-    job.job_result_uri = "https://" + space_name + ".crosschain.computer"
+    wallet_address = job.job_source_uri.split('/')[-2]
+    task = build_space_task.delay(space_name, wallet_address)
+    app_name = f"{space_name}-{wallet_address}"
+    job.job_result_uri = f"https://{app_name}.crosschain.computer"
```

Comment:
I think this is limited by BQ's version?
on local it might not work

Reply:
Yeah, the result uri was hard-coded. It should be in the env file. @flin-nbai  would you mind helping me fix it?

Reply:
What should it be changed to? It should match the host_name for the k8s ingress right?

Reply:
I mean we can move the domain name to env file or config file

Reply:
updated